docs/src/components/SideNav.tsx

```diff
@@ -68,9 +68,18 @@ export function List({ className, ...props }: React.ComponentProps<'ul'>) {
   return <ul className={clsx('SideNavList', className)} {...props} />;
 }
 
+export function Label({ className, ...props }: React.ComponentProps<'span'>) {
+  return <span className={clsx('SideNavLabel', className)} {...props} />;
+}
+
+export function Badge({ className, ...props }: React.ComponentProps<'span'>) {
+  return <span className={clsx('SideNavBadge', className)} {...props} />;
+}
+
```

Comment:
In line with the style/perf changes I've been doing, I'd update the style for these to this more performant and terser version:

```suggestion
export function Label(props: React.ComponentProps<'span'>) {
  return <span {...props} className={clsx('SideNavLabel', props.className)} />;
}

export function Badge(props: React.ComponentProps<'span'>) {
  return <span {...props} className={clsx('SideNavBadge', props.className)} />;
}

```

Reply:
Thanks updated ~ I suppose we should do this for all the docs components then